src/main/java/by/exadel/internship/entity/User.java

```diff
@@ -1,20 +1,22 @@
-package by.exadel.internship.entity.user;
-import by.exadel.internship.dto.enums.Technology;
+package by.exadel.internship.entity;
+
+import by.exadel.internship.dto.enums.Skill;
 import by.exadel.internship.dto.enums.UserRole;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+
 import javax.persistence.*;
-import java.util.List;
+import java.util.Set;
 import java.util.UUID;
 
 @Entity
-@Table(name = "user")
+@Table(name="\"user\"")
```

Comment:
I think it's not a good idea to wrap with quotes. This problem had to be appeared when the entity was created (probably it's not tested).
My suggestion to rename table to `user_detail`.
@viktarkolbik @SergeyKoval what do you think?